site2/docs/administration-geo.md

```diff
@@ -12,24 +12,24 @@ The diagram below illustrates the process of geo-replication across Pulsar clust
 
 ![Replication Diagram](assets/geo-replication.png)
 
-In this diagram, whenever producers **P1**, **P2**, and **P3** publish messages to the topic **T1** on clusters **Cluster-A**, **Cluster-B**, and **Cluster-C**, respectively, those messages are instantly replicated across clusters. Once replicated, consumers **C1** and **C2** can consume those messages from their respective clusters.
+In this diagram, whenever **P1**, **P2**, and **P3** producers publish messages to the **T1** topic on **Cluster-A**, **Cluster-B**, and **Cluster-C** clusters respectively, those messages are instantly replicated across clusters. Once replicated, **C1** and **C2** consumers can consume those messages from their respective clusters.
```

Comment:
Since in the example P1, P2 and P3 are the name of the topics and not adjectives, isn't the current form correct?

Reply:
@merlimat Thank you very much for raising this question. P1, P2, and P3 are the name of the topics, indeed, they are not adjectives. Here i treat it as proper noun or keyword.
I will adopt "the xx producer", "the xx consumer", "the xx topic", "the xx command" patterns in writing if you don't object it. In this pattern, "xx" is the name of producer/consumer/topic/command, it does not act as adjective, but still as noun.  We can regard producer/consumer/topic/command as proper nouns or keywords.
Such style derives from IBM style guide:
1. When you refer to a command, parameter, or option in running text, specify the word command, parameter, or option after the name. 
- For example: You can use the db2trc command to ...
2. Specify the keyword type after the keyword. 
- For example: Create a message producer by calling the messageProducer method.
3. Avoid placing the noun *file* or *directory* before the name of the file or directory. Instead, use the name as an adjective, followed by the noun file or directory.  
- For example: View the readme.txt file to determine the version level.
...
If you have different opinion on this, feel free to let me know.